subscriptions/templates/subscriptions/subscription_detail.html

```diff
@@ -0,0 +1,110 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Subscription" %}{% endblock %}
+
+{% block content %}
+    {% include 'account/_logged_in_sidebar.html' %}
+    <section class="account-form">
+        <h2>{% trans "Subscription" %}</h2>
+        {% include 'account/_messages.html' %}
+
+        {% if quota_status.blocked %}
+          {% if quota_status.count > quota_status.quota %}
+            <p class="attention-box warning">
+                You have used up your quota for the month.
+                Please <a href="{% url "subscription_update" %}">upgrade</a>
+                or contact us.
+            </p>
+          {% else %}
+            <p class="attention-box warning">
+                Your account is currently suspended. Please contact us.
+            </p>
+          {% endif %}
+        {% endif %}
+
+        {% if stripe.plan %}
+
+            <p>Your current plan is <strong>{{ stripe.plan.name }}</strong> ({{ stripe.plan.id }}).</p>
+
+            <p>It costs you £{{ actual_paid|floatformat:-2 }}/mth.
+            {% if stripe.discount %}
+                (£{{ stripe.plan.amount|floatformat:-2 }}/mth with
+                {{ stripe.discount.coupon.percent_off }}% charitable discount.)
```

Comment:
At first glance this wording confused me a bit. Something like "(£20/mth with 100% charitable discount **applied**.)" would make it clearer to me.